config/main.py

```diff
@@ -614,6 +614,91 @@ def remove(session_name):
     config_db.connect()
     config_db.set_entry("MIRROR_SESSION", session_name, None)
 
+#
```

Comment:
do we also have a show command that should be added as well?

Reply:
yes, show is also updated in this PR